services/offline/src/lib/__tests__/network-status.test.tsx

```diff
@@ -109,9 +110,11 @@ describe('debouncing state changes', () => {
         // Immediately, nothing should happen
         expect(result.current.online).toBe(true)
 
+        await wait(1000)
```

Comment:
I had to add a couple of explict waits here in place of the previous timeouts .. I think this is equivalent of the previous tests, but maybe we can rewrite the tests with fake timers or something in the near future